### PR TITLE
Refactor explore.spec.ts

### PR DESCRIPTION
**Pull request by nick-dev123**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [X] Refactoring

## Context

- [X] Related to #7051
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [X] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?usdamount=20&target=12ZLwasAvpdxDiiMedr11VeKUyQkY3WfneMkswBCAHVvxAqe)

#### Community participation

- [ ] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at ccf9e6d</samp>

This pull request refactors several pages to use the composition API and the `useHead` composable from `@vueuse/head` library. This improves the code readability and consistency, and simplifies the document head management. The pages affected are `pages/e2e-login.vue`, `pages/privacy-policy.vue`, `pages/series-insight.vue`, `pages/carbonless.vue`, `pages/cookie-notice.vue`, `pages/hot.vue`, `pages/qrCode.vue`, and `pages/sales.vue`.

### <samp>🤖 Generated by Copilot at ccf9e6d</samp>

> _To manage the document head_
> _They used `useHead` instead_
> _Of the old `head` method_
> _That was not so splendid_
> _And refactored some pages they said_

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -73,5 +73,6 @@ export default defineConfig({
     command: process.env.CI ? 'pnpm start:static' : 'pnpm run dev',
     url: 'http://localhost:9090',
     reuseExistingServer: !process.env.CI,
+    timeout: 2 * 60 * 1000,
```
Comment (Jarsen136) [inline playwright.config.ts:76]: Is it necessary to add such a long timeout?

**Comment by prury** [comment]
oh, nice! thanks! 
i may be wrong, but running it with --trace on locally only validates that the skeletons are there, but not verifying if the images itself are loading
the global timeout i would leave it like i did on #7267 
can we replace regex with something more tangible? since Sonar Cloud is reporting Security Hotspot

File: playwright.config.ts (hunk 1 above)
Comment (nick-dev123) [reply playwright.config.ts:76]: The default is 1 minute and it failed to build it on my computer. I can remove if it's not necessary.

Comment (Jarsen136) [reply playwright.config.ts:76]: OK. Then let's keep it.

**Review by Jarsen136 — APPROVED** [review]

**Comment by roiLeo** [comment]
> playwright not passing because of redirect test that i poorly refactored, but already made some changes on another PR

which one?

**Comment by prury** [comment]
> > playwright not passing because of redirect test that i poorly refactored, but already made some changes on another PR
> 
> which one?

yeah, now many are failing, i see it, previously was only redirect.spec.ts

**Comment by yangwao** [comment]
pay 30 usd

**Comment by nick-dev123** [comment]
@yangwao The payment didn't go through

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.03 USD/DOT ~ 7.444 $DOT
🧗 [12ZLwasAvpdxDiiMedr11VeKUyQkY3WfneMkswBCAHVvxAqe](https://polkadot.subscan.io/account/12ZLwasAvpdxDiiMedr11VeKUyQkY3WfneMkswBCAHVvxAqe)
🔗 [0x60d3bb900e13ddbfde340b1d506df6aedf499f197ac254f4f986d56568dde0b7](https://polkadot.subscan.io/extrinsic/0x60d3bb900e13ddbfde340b1d506df6aedf499f197ac254f4f986d56568dde0b7)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
> @yangwao The payment didn't go through

https://github.com/kodadot/nft-gallery/issues/7356

**Pull request merged**